package.json

```diff
@@ -24,6 +24,8 @@
   },
   "name": "workspace-library",
   "dependencies": {
+    "@types/styled-components": "^5.0.1",
+    "react-native-webview-quilljs": "^2.0.0",
```

Comment:
Root package.json can't have any dependency, you can check that WebViewQuillJS was added as a package folder and we want all our libraries in node_modules. Please remove this and add them on the package.json at mobile.

You can do this from the root folder like so:

` bash ./workspace.sh mobile add react-native-webview-quilljs`
`bash ./workspace.sh mobile add @types/styled-components -D`

Note that I used a -D on @types/styled-components since its a devDependency